src/jdk.jdwp.agent/unix/native/libjdwp/exec_md.c

```diff
@@ -44,6 +48,104 @@ static char *skipNonWhitespace(char *p) {
     return p;
 }
 
+int
+isAsciiDigit(char c)
+{
+  return c >= '0' && c <= '9';
+}
+
+#if defined(_AIX)
+  /* AIX does not understand '/proc/self' - it requires the real process ID */
+  #define FD_DIR aix_fd_dir
+  #define DIR DIR64
+  #define dirent dirent64
+  #define opendir opendir64
+  #define readdir readdir64
+  #define closedir closedir64
+#elif defined(_ALLBSD_SOURCE)
+  #define FD_DIR "/dev/fd"
+#else
+  #define FD_DIR "/proc/self/fd"
+#endif
+
+// closes every file descriptor that is listed as a directory
+// entry in "/proc/self/fd" (or its equivalent). standard
+// input/output/error file descriptors will not be closed
+// by this function. this function returns 0 on failure
+// and 1 on success
```

Comment:
Pre-existing nit: Please start sentences with a capital and end with a period. Thanks. This applies through the copied code.

Reply:
I've fixed the code comments to follow this suggestion. I've left single sentence comments as is. If there's a preference to change them too, do let me know and I'll update those too.